backend/src/main/java/de/neuefische/backend/model/CounselingCenter.java

```diff
@@ -0,0 +1,23 @@
+package de.neuefische.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class CounselingCenter {
+    private String name;
```

Comment:
i would add a unique id field.
A unique id makes it easier to access a specific Center and you can also use the id as key in your frontend.